Specifications/catalog/catalog.binding.https.md

```diff
@@ -0,0 +1,134 @@
+# Catalog HTTPS Binding
+
+## 1 Introduction
+
+This specification defines a RESTful API over HTTPS for the [Catalog Protocol].
+
+The OpenAPI definitions for this specification can be accessed [here](TBD).
+
+## 2 Path Bindings
+
+### 2.1 Prerequisites
+
+1. The `<base>` notation indicates the base URL for a catalog service endpoint. For example, if the base catalog URL is `api.example.com`, the URL `https://<base>/catalog/request`
+   will map to `https//api.example.com/catalog/request`.
+
+2. All request and response messages must use the `application/json` media type.
+
+### 2.2 CatalogErrorMessage
+
+In the event of a request error, the catalog service must return an appropriate HTTP code and a [CatalogErrorMessage](./catalog.protocol.md#) in the response body.
+
+| Field   | Type          | Description                                                 |
+|---------|---------------|-------------------------------------------------------------|
+| code    | string        | An optional implementation-specific error code.             |
+| reasons | Array[object] | An optional array of implementation-specific error objects. |
+
+### 2.3 The `catalog/request` endpoint
+
+#### 2.3.1 POST
+
+The [CatalogRequestMessage](catalog.protocol.md#1-catalogrequestmessage) corresponds to `POST https://<base>/catalog/request`:
+
+```
+POST https://provider.com/catalog/request
```

Comment:
Why not GET?

Reply:
Because of the need for filters and query expressions. These become very difficult (and potentially unsafe) to put in URL params.